D2. Requirements.md

```diff
@@ -0,0 +1,435 @@
+
+# CS 386 – Software Engineering
+## **Team Project – D.2 Requirements**
+
+Grading: 30 points
+In this deliverable, you will report the results for the requirements engineering for your product. Structure your deliverable using the following sections. 
+
+**Positioning**
+## **Problem statement**
+Provide a statement to summarize the problem solved by your project according to the following structure (which is based on the OpenUP Vision Template):
+
+Example of a problem statement: “The problem of inefficient prioritization of homework assignments affects college students; the impact of which is delivering 
```

Comment:
Please, remove these parts of the template like "Example of a problem statement: " and leave just your answer

Reply:
do the same for the other items as well